### PR TITLE
Add a Grunt Connect Task. Easy browser access to nice examples

### PR DESCRIPTION
**Pull request by gintelllect**

### CODE REVIEW

**Comment by katspaugh** [comment]
There's something wrong with the formatting.

**Comment by gintelllect** [comment]
Fixed the formatting. Please check.

File: Gruntfile.js [hunk 1]
```diff
@@ -27,6 +27,15 @@ module.exports = function(grunt) {
...
       }
     },
+    connect: {
+        options: {
+	    base: '.',
+	    port: 9000,
+	    // Change this to '0.0.0.0' to access the server from outside.
+	    hostname: 'localhost',
+	    keepalive: true
+	    },
```
Comment (katspaugh) [inline Gruntfile.js]: This brace should be outdented.

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "grunt-contrib-uglify": ">=0.5.0",
     "grunt-contrib-jshint": ">=0.10.0",
     "grunt-contrib-jasmine": ">=0.8.2",
+    "grunt-contrib-connect": "~0.3.0",
```
Comment (katspaugh) [inline package.json]: Please use `npm install --save-dev grunt-contrib-connect` to place the dependency in the correct place (sorted alphabetically).

**Comment by katspaugh** [comment]
Thanks!

**Pull request merged**